### PR TITLE
sql seed script

### PR DESCRIPTION
**Pull request by jennthh**

### CODE REVIEW

File: stock-price-points/data/dataGenerator.js [hunk 1]
```diff
@@ -133,9 +134,39 @@ function assembleTestData(set, wstream, label) {
...
+  wstream.write('companyAbbriev,company,weeks,yearly,currentPrice\n');
+
+  tickers.forEach((ticker, index) => {
+    let companyAbbriev = ticker;
+    let company = generateCompanyNames(ticker);
+    let weeks = generateDataDist();
+    let yearly = generateAnnualData(weeks);
+    let currentPrice = generateCurrentPrice(yearly.yearAverage);
+    // let dataEntry = { company, companyAbbriev, weeks, yearly, currentPrice };
+    let dataEntry = [companyAbbriev, company, '"' + JSON.stringify(weeks) + '"', '"' + JSON.stringify(yearly) + '"', '"' + JSON.stringify(currentPrice) + '"']
```
Comment (eddielo91) [inline stock-price-points/data/dataGenerator.js:151]: Maybe use backticks with ${}. IMO it looks better, but its personal preference here.

File: stock-price-points/database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  -- istockprices,
+  -- jstockprices,
+  -- kstockprices,
+  -- lstockprices,
+  -- mstockprices,
+  -- nstockprices,
+  -- ostockprices,
+  -- pstockprices,
+  -- qstockprices,
+  rstockprices,
```
Comment (eddielo91) [inline stock-price-points/database/schema.sql:27]: Do you have a table for each of them???

**Review by eddielo91 — APPROVED** [review]
Everything's good. Just preferential use of ES6 but nothing causing issues.

File: stock-price-points/database/schema.sql (hunk 2 above)
Comment (jennthh) [reply stock-price-points/database/schema.sql:27]: yeah TT__TT in the attempt to make my query times shorter. i'll combine them eventually

**Pull request merged**